src/app/pages/Sandbox/Editor/Content/Header/ShareView.js

```diff
@@ -296,11 +343,25 @@ class ShareView extends React.PureComponent {
                       setValue={this.setHideNavigation}
                     />
                     <PaddedPreference
+                      title="Single file entry script"
```

Comment:
Maybe we should make this title some more clear, something like "Show selected module". Still not very descriptive, but maybe a bit better. It's very hard to describe this in one line.

We could also change this preference to the actual switch button from the preview, like we do for the view modes. Then the ui explains the functionality.